[BUG] Botkube 0.8.0 - crash loop - invalid memory address or nil pointer deference

**Describe the bug**
I have tried to deploy Botkube version 0.8.0 (which was recently released), I am using the deploy-all-in-one-tls.yaml to deploy it in my Kubernetes cluster. 

**To Reproduce**
Steps to reproduce the behavior:
1. I am running mattermost server version: 5.11.0
2. Kubebot version 0.8.0
3. Deploy the deploy-all-in-one-tls.yaml with TLS (self-signed certificates)
4. See error:

``` 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x10a8e3c] 

goroutine 78 [running]:
github.com/infracloudio/botkube/pkg/bot.(*mmBot).Start.func1(0xc00010e150, 0xc0002af140) 
       /go/src/github.com/infracloudio/botkube/pkg/bot/mattermost.go:104 +0x5c 
created by github.com/infracloudio/botkube/pkg/bot.(*mmBot).Start 
      /go/src/github.com/infracloudio/botkube/pkg/bot/mattermost.go:101 +0x546​
```

--- a/pkg/bot/mattermost.go
+++ b/pkg/bot/mattermost.go
@@ -101,7 +101,7 @@ func (b *MMBot) Start() {
 	go func() {
 		for {
 			event := <-webSocketClient.EventChannel
-			if event.Event != model.WEBSOCKET_EVENT_POSTED {
+			if event == nil || event.Event != model.WEBSOCKET_EVENT_POSTED {
 				continue
 			}
 			post := model.PostFromJson(strings.NewReader(event.Data["post"].(string)))

